src/test/java/org/jabref/gui/ClipBoardManagerTest.java

```diff
@@ -1,70 +0,0 @@
-package org.jabref.gui;
-
-import java.awt.datatransfer.Clipboard;
-import java.awt.datatransfer.DataFlavor;
-import java.awt.datatransfer.Transferable;
-import java.util.Arrays;
-import java.util.List;
-
-import org.jabref.logic.importer.ImportException;
-import org.jabref.logic.importer.ImportFormatReader;
-import org.jabref.logic.importer.ImportFormatReader.UnknownFormatImport;
-import org.jabref.logic.importer.ParserResult;
-import org.jabref.model.entry.BibEntry;
-
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.mockito.ArgumentMatchers;
-
-import static org.junit.Assert.assertEquals;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-
-public class ClipBoardManagerTest {
-
-    private ClipBoardManager clipBoardManager;
```

Comment:
What about the tests?

Reply:
The clipboard manager needs to run on the JavaFX thread to work, which is not easily achieved in a test. Thus, deleting these tests was easier than trying to rewrite them ;-)